Update specter docs (#394)
Start9Labs/documentation: site/source/user-manual/service-guides/specter/specter-service.rst

.. _specter-service:

====================

> Specter as a Service

Reviewer: ```suggestion
Specter Server
```

Let's avoid "as a service" - it has connotations

Author: I agree, the tradeoff here though is that I have to be very clear about when I'm referring to Specter running on the server vs Specter Desktop. It's a confusing setup - I'm going to research if the HWI can run as a daemon on the client device without using Specter Desktop now.

                :width: 60%
                :alt: cc-name-wallet

            .. tip:: If you only have one key, it will automatically use that key. You cannot select a configuration that doesn't match the available derivation path(s).

        #. Select **Scan for existing funds** if you have already used this wallet and wish to establish the transaction history, if this is a brand new wallet this is not necessary and should be deselected:

            .. figure:: /_static/images/services/specter/cc-scan.png
                :width: 60%
                :alt: cc-scan

            .. note:: Rescanning will be very slow if Specter is configured to connect to Bitcoin Core directly, and extremely fast if connecting via Electrs.

        #. Click **Create wallet**:
        
            .. figure:: /_static/images/services/specter/cc-click-create-wallet.png
                :width: 60%
                :alt: cc-click-create-wallet

        #. Specter is now setup to use your air gapped signing device!

            .. figure:: /_static/images/services/specter/cc-complete.png
                :width: 60%
                :alt: cc-complete

    .. group-tab:: USB Connected Signing Devices:

>         For devices like the Trezor One on which air gapping is not possible xpubs must be imported via USB - Specter permits this via the HWI which requires running a second instance of Specter on your *local* machine (i.e *not* your server).

Reviewer: ```suggestion
        For devices that do not support air-gap features, such as the Trezor One, xpubs must be imported via USB - Specter permits this via the HWI which requires running a second instance of Specter on your *local* machine (i.e *not* your server).
```
